### PR TITLE
add param choices for sirius

### PR DESCRIPTION
**Pull request by anupriyatripathi**
added choices for instrument and CSI finger ID database

### CODE REVIEW

File: q2_chemistree/plugin_setup.py [hunk 1]
```diff
@@ -29,17 +29,16 @@
...
-    'database': Str % Choices(['all', 'PubChem', 'bio']),
+    'database': Str % Choices(['all', 'pubchem']),
     'sirpath': Str,
-    'instrument': Str,
-    'dbcsi': Str,
-    'ppmlim': Int,
+    'instrument': Str % Choices(['qtof', 'orbitrap', 'fticr']),
+    'dbcsi': Str % Choices(['all', 'pubchem', 'bio', 'kegg', 'hmdb']),
+    'ppmlim': Int % Range(0, 30, inclusive_end=True),
     'nproc': Int,
```
Comment (ElDeveloper) [inline q2_chemistree/plugin_setup.py]: Can you add a range for the number of processors, and rename the parameter to match other plugins in the qiime2 ecosystem. See for example q2-diversity or q2-feature-classifier.

File: q2_chemistree/plugin_setup.py [hunk 2]
```diff
@@ -29,17 +29,16 @@
                                         artifact_format=MGFDirFmt)
 
 PARAMS = {
-    'database': Str % Choices(['all', 'PubChem', 'bio']),
+    'database': Str % Choices(['all', 'pubchem']),
```
Comment (ElDeveloper) [inline q2_chemistree/plugin_setup.py]: In general it is better if these parameters can be more verbose about what they are. Abbreviations and contractions are often hard to decipher for new users. Unless it is a universal abbreviation, I suggest removing it and spelling out the parameter names. See for example the suggestions in the scikit-bio coding guidelines:

http://scikit-bio.org/docs/latest/development/coding_guidelines.html#acceptable-abbreviations

File: q2_chemistree/plugin_setup.py [hunk 3]
```diff
@@ -31,8 +31,8 @@
 PARAMS = {
     'database': Str % Choices(['all', 'PubChem', 'bio']),
     'sirpath': Str,
-    'instrument': Str,
-    'dbcsi': Str,
+    'instrument': Str % Choices(['qtof', 'orbitrap', 'fticr']),
+    'dbcsi': Str % Choices(['PubChem', 'bio']),
```
[the remark this answers is not in the document]
Comment (anupriyatripathi) [reply q2_chemistree/plugin_setup.py]: It's a great point - I had to double check. Looks like one could. PubChem would be a superset so it'll work. We've updated these choices in the new PR to make sure that each combination works.

File: q2_chemistree/plugin_setup.py (hunk 1 above)
Comment (anupriyatripathi) [reply q2_chemistree/plugin_setup.py]: Sure, thanks for the suggestion! Let's use the same parameter names as SIRIUS uses? They are fairly descriptive.

Comment (ElDeveloper) [reply q2_chemistree/plugin_setup.py]: That sounds like a good idea granted that they are no redundant or out of context 👍

**Pull request merged**